docs/transfers.rst

```diff
@@ -0,0 +1,296 @@
+Nodes validate the proof-of-work of a transaction by calculating the transaction's
+hash from the attached transaction trytes. If the resulting hash has at least
+``minimum weight magnitude`` number of trailing zero trits, it is correct.
+
+In PyOTA, use :py:meth:`~Iota.attach_to_tangle` to carry out this step.
+
+5. Broadcast and Store
+~~~~~~~~~~~~~~~~~~~~~~
+
+The final step is to send the bundle to the network. Nodes will broadcast
+the transactions in the network, and store them in their local database.
+
+In PyOTA, use :py:meth:`~Iota.broadcast_and_store` to achieve this.
+
+Observe the bird-eye view of the Tangle depicted at the last step of the
+process. Our transactions are part of the Tangle, reference each other and
+the two tips. Newer transactions may reference our transactions as branch or
+trunk.
+
+Use the Library
+---------------
+
+The IOTA libraries help you to abstract away the low-level operations needed
+to create transfers. The figure below illustrates the different ways you can
+build and send a transfer.
+
+.. figure:: images/transfer_api.svg
+   :scale: 100 %
+   :alt: Different ways of sending a transfer in IOTA.
+
+   API commands for sending transfers.
+
+Let's look at some code snippets on how to perform the above with an imaginary
+bundle that has 3 fictional transactions.
+
+1. Level Padawan
+~~~~~~~~~~~~~~~~
+The easiest and most convenient way is to use :py:meth:`~Iota.send_transfer`
+extended API method. You still need to create the transactions yourself
+with :py:class:`ProposedTransaction`.
+
+.. code-block::
+
+    from iota import Iota, ProposedTransaction, Address
+
+    api = Iota('https://nodes.devnet.iota.org:443')
+
+    fictional_transactions = [
+        ProposedTransaction(
+                address=Address(b'FIRSTRANDOMADDRESS'),
+                value=0,
+                # You could add a tag or message here too!
+        ),
+        ProposedTransaction(
+                address=Address(b'SECONDRANDOMADDRESS'),
+                value=0,
+        ),
+        ProposedTransaction(
+                address=Address(b'THIRDRANDOMADDRESS'),
+                value=0,
+        )
+    ]
+
+    imaginary_bundle = api.send_transfer(
+        transfers=transactions
+    )['bundle']
+
+As all API methods in PyOTA, :py:meth:`~Iota.send_transfer` also returns
+a ``dict``. The ``bundle`` key holds the value of :py:class:`Bundle`.
+
+It's important to note, that for value transfers, you will need your seed as well.
+:py:meth:`~Iota.send_transfer` will look for ``input addresses`` to fund outgoing
+transactions in the bundle, and auto-generate an unused ``change address`` if
+there is a remainder amount of tokens. It will also take care of finalizing the
+bundle and signing the necessary input transactions.
+
+2. Level Obi-Wan
+~~~~~~~~~~~~~~~~
+Instead of :py:meth:`~Iota.send_transfer`, you can use the combination of
+:py:meth:`~Iota.prepare_transfer` and :py:meth:`~Iota.send_trytes` to achieve
+the same result.
```

Comment:
```suggestion
the same result.

.. tip::
  This can be useful if you want to prepare the transactions (including signing inputs) on one device, but you want to then transfer the data to another device for transmission to the Tangle.  For example, you might :py:meth:`~Iota.prepare_transfer` on an air-gapped computer that has your seed stored on it, but then transfer the resulting trytes to a networked computer (that does not have your seed) to :py:meth:`~Iota.send_trytes`.
```